### PR TITLE
Updated configuration.md

### PR DESCRIPTION
**Pull request by Rajitha20H**

### CODE REVIEW

File: _includes/configuration.md [hunk 1]
```diff
@@ -1,14 +1,14 @@
 ## Configuring {{ site.data.product.title_short }}
 
-After installing {{ site.data.product.title_short }} and running it for the first
+After installing {{ site.data.product.title_short }} and run it for the first
```
Comment (Fryguy) [inline _includes/configuration.md:3]: I think running is more grammatically correct.

File: _includes/configuration.md [hunk 2]
```diff
@@ -43,9 +43,6 @@ options for database configuration:
 
   - Configure the appliance to use an external PostgreSQL database
 
-#### Configuring an Internal Database
-
-{% include configuration-db.md %}
```
Comment (Fryguy) [inline _includes/configuration.md:48]: This should probably stay, but we can move the `#### Configuring an Internal Database` inside the included file.  That way you can exclude it downstream if you need to.

File: _includes/configuration.md [hunk 3]
```diff
@@ -74,7 +70,7 @@ necessary to alter the `max_connections` setting.
 
 **Note:**
 
-  - {{ site.data.product.title_short }} requires PostgreSQL version 9.5.
+  - {{ site.data.product.title_short }} requires PostgreSQL version 10.
```
Comment (Fryguy) [inline _includes/configuration.md:73]: Nice find.

**Comment by Fryguy** [comment]
@bdunne Please also review.

File: _includes/configuration.md (hunk 1 above)
Comment (kbrock) [reply _includes/configuration.md:3]: "parallelism" with `installing`

**Comment by kbrock** [comment]
Thanks @Rajitha20H - Good stuff

File: _includes/configuration.md [hunk 4]
```diff
@@ -60,8 +57,7 @@ database.
 The `postgresql.conf` file used with {{ site.data.product.title_short }} databases requires
 specific settings for correct operation. For example, it must correctly
 reclaim table space, control session timeouts, and format the PostgreSQL
-server log for improved system support. Due to these requirements, Red
-Hat recommends that external {{ site.data.product.title_short }} databases use a
+server log for improved system support. It is recommended that external {{ site.data.product.title_short }} databases use a
```
Comment (bdunne) [inline _includes/configuration.md:60]: I think we can remove the `title_short` reference completely for simplicity

Comment (Rajitha20H) [reply _includes/configuration.md]: @bdunne  All the instances has been replaced with ManageIQ.

Comment (Fryguy) [reply _includes/configuration.md]: @Rajitha20H Switching to ManageIQ is not what @bdunne was suggesting.  Those values were set to `{{ site.data.product.title_short }}` because we they need to be changed between upstream and downstream, so ideally we shouldn't use the name `ManageIQ` anywhere in the docs, but instead use `{{ site.data.product.title_short }}`

What @bdunne was suggesting was that in this particular instance, having the product name doesn't provide any more information to the reader, so we can remove the product name entirely.

**Comment by Fryguy** [comment]
@Rajitha20H The change to ManageIQ should be undone... See https://github.com/ManageIQ/manageiq-documentation/pull/1641#discussion_r803723506

**Comment by Rajitha20H** [comment]
@Fryguy, all the changes to ManageIQ have been undone, back to {{ site.data.product.title_short }}. The specific reference to {{ site.data.product.title_short }} in the sentence also is removed as suggested by @bdunne. Hope it is ok.

File: _includes/configuration.md (hunk 1 above)
Comment (bdunne) [reply _includes/configuration.md:3]:
```
After installing and running it for the first time, some basic configuration is required.
You must complete the following steps:
```

File: _includes/configuration.md [hunk 5]
```diff
@@ -1,14 +1,14 @@
...
+{{ site.data.product.title_short }}, you must complete these steps:
 
-1.  Add a disk to the infrastructure hosting your appliance.
+1.  Add a disk to the infrastructure that is hosting your appliance.
 
 2.  Configure the database.
 
-Configure the {{ site.data.product.title_short }} appliance using the internal
+Configure the {{ site.data.product.title_short }} appliance by using the internal
 appliance console.
```
Comment (bdunne) [inline _includes/configuration.md:12]:
```suggestion
Configure the appliance by using the internal appliance console.
```

Comment (Fryguy) [reply _includes/configuration.md:12]: @bdunne Since the title_short was there previously, I don't think this suggestion should hold up a merge.  WDYT?

File: _includes/configuration.md [hunk 6]
```diff
@@ -90,32 +86,18 @@ necessary to alter the `max_connections` setting.
 
 4.  Select **Configure Application** from the menu.
 
-5.  You are prompted to create or fetch a security key.
```
Comment (bdunne) [inline _includes/configuration.md:93]: Why is this removed?

File: deleting_a_router.md [hunk 7]
```diff
@@ -1,10 +1,10 @@
-### Deleting a Network Router
+
```
Comment (bdunne) [inline deleting_a_router.md:1]: Extra newline?

File: installing_on_vmware_vsphere/_topics/installation.md [hunk 8]
```diff
@@ -58,7 +58,7 @@ Use the following procedure to upload the {{ site.data.product.title_short }} ap
 5.  Select the host or cluster on which you want to deploy the OVF
     template and click **Next**.
 
-6.  Select the host on which you want to run the run the {{ site.data.product.title_short }}
+6.  Select the host on which you want to run the {{ site.data.product.title_short }}
     appliance, and click **Next**.
```
Comment (bdunne) [inline installing_on_vmware_vsphere/_topics/installation.md:62]:
```suggestion
6.  Select the host on which you want to run the appliance, and click **Next**.
```

File: _includes/configuration.md (hunk 5 above)
Comment (bdunne) [reply _includes/configuration.md:12]: No, this alone shouldn't hold up a merge

File: _includes/configuration.md (hunk 1 above)
Comment (Rajitha20H) [reply _includes/configuration.md:3]: Updated.

File: _includes/configuration.md (hunk 2 above)
Comment (Rajitha20H) [reply _includes/configuration.md:48]: It is already in the includes folder.

File: _includes/configuration.md (hunk 6 above)
Comment (Rajitha20H) [reply _includes/configuration.md:93]: It is removed as per @Fryguy 's comments in https://github.com/ManageIQ/manageiq-documentation/pull/1626#

File: _includes/configuration.md [hunk 9]
```diff
@@ -90,32 +85,18 @@ necessary to alter the `max_connections` setting.
...
-      - If this is not the first {{ site.data.product.title_short }} appliance, choose
-        **Fetch key from remote machine** to fetch the key from the
-        first appliance.
-
-        **Note:**
-
-        All {{ site.data.product.title_short }} appliances in a multi-region
-        deployment must use the same key.
-
 6.  Choose **Create Region in External Database** for the database location.
```
Comment (bdunne) [inline _includes/configuration.md:88]:
```suggestion
5.  Choose **Create Region in External Database** for the database location.
```

File: _includes/configuration.md [hunk 10]
```diff
@@ -90,32 +85,18 @@ necessary to alter the `max_connections` setting.
...
 
 7.  Enter the database hostname or IP address when prompted.
```
Comment (bdunne) [inline _includes/configuration.md:90]:
```suggestion
6.  Enter the database hostname or IP address when prompted.
```

File: deleting_a_router.md [hunk 11]
```diff
@@ -1,10 +1,10 @@
...
+# Deleting a Network Router
```
Comment (bdunne) [inline deleting_a_router.md:2]:
```suggestion
# Deleting a Network Router
```

Comment (Fryguy) [reply deleting_a_router.md:2]: I think this change was pulled out into a separate PR.

Comment (bdunne) [reply deleting_a_router.md:2]: Ah, I didn't see the merge conflict

**Comment by Fryguy** [comment]
@Rajitha20H This is conflicted by the merge of #1646
